guides/advanced_guides/field_properties.md

```diff
@@ -1,36 +1,67 @@
+#### Example
 
-This list can be restricted to a specific set of attributes that you can select in the settings. That way, you are able to specify which fields should be ignored by MeiliSearch from searches.
+Suppose you manage a database that contains information about movies. By adding the following settings, the fields `uid`, `movie_id`, `title`, `description`, `poster`, `release_date` and `rank` will be searched.
 
-## Displayed attributes
+```bash
+$ curl \
+  -X POST 'http://localhost:7700/indexes/movies/settings' \
+  --data '{
+      "searchableAttributes": [
+          "uid",
+          "movie_id",
+          "title",
+          "description",
+          "poster",
+          "release_date",
+          "rank"
+      ]
+  }'
+```
 
-Displayed fields are fields whose attribute is present in the [displayed-attributes](/references/displayed_attributes.md) list.
+## Displayed Fields
 
-Documents returned upon search **contain only displayed fields**.
+The fields whose attributes are added to the [displayed-attributes list](/references/displayed_attributes.md) are **displayed in each matching document**.
 
-Therefore, when a field's attribute is not in the displayed-attribute list, the field won't be added to the documents.
+Documents returned upon search contain only displayed fields.
 
-By default, all field attributes are added to the displayed-attributes list. If a new document is ingested and one of its fields is missing from other documents, the latter will be automatically added to the displayed-attributes list. [This behavior can be changed](/references/accept_new_fields.md).
+**By default, all field attributes are added to the displayed-attributes list**. If a new document is ingested and one of its fields is missing from other documents, the latter will be automatically added to the displayed-attributes list. [This behavior can be changed](/references/accept_new_fields.md).
 
-This list can be restricted to a specific set of attributes that you can select in the settings. That way, you are able to specify which fields should not be returned upon search.
+Therefore, if a field attribute is not in the displayed-attribute list, the field won't be added to the documents but will remain stored in the server. This list can be restricted to a selected set of attributes in the settings.
+
+#### Example
+
+Suppose you manage a database that contains information about movies. By adding the following settings, documents returned upon search will contain the fields `title`, `description`, `poster`, `release_date` and `rank`.
+
+```bash
+$ curl \
+  -X POST 'http://localhost:7700/indexes/movies/settings' \
+  --data '{
+      "displayedAttributes": [
+          "title",
+          "description",
+          "poster",
+          "release_date",
+          "rank"
+      ]
+  }'
+```
 
 ## Data storing
 
-All fields are stored. This behavior **cannot be changed**.
+All fields are stored. **This behavior cannot be changed**.
```

Comment:
Once we have accepted the PR of Thomas we should link it here.

Reply:
Noted!